### ISSUE TITLE
WSL app name is not displayed on startup menu

### ISSUE THREAD

**Issue by Luberen** [Microsoft/WSL#3481]
**Please fill out the below information:**
* Your Windows build number:  17134.228

![debian](https://user-images.githubusercontent.com/26616056/44304329-28cfe280-a394-11e8-88bf-4fcfd721c86d.png)

I've installed Debain GNU/Linux but the app name is not displayed on startup menu.

**Comment by Voskrese** [comment]
WSL тут причем? bash не запускается?

**Comment by benhillis** [comment]
@Luberen - Is Debian the only WSL distro that the name doesn't appear?

**Comment by benhillis** [comment]
@Luberen - Thanks! @rhaist, it seems that the new version of Debian does not have a title when the app is pinned to the start menu.  Is this something you are aware of?

**Comment by rhaist** [comment]
@benhillis thanks for the hint. This is probably a bug. I'll investigate.

**Comment by Biswa96** [comment]
@rhaist In new commits, you removed these lines from `MyDistro.appxmanifest` or `AppxManifest.xml` file:

```xml
<uap:DefaultTile>
    <uap:ShowNameOnTiles>
        <uap:ShowOn Tile="square150x150Logo" />
        <uap:ShowOn Tile="wide310x150Logo" />
        <uap:ShowOn Tile="square310x310Logo" />
    </uap:ShowNameOnTiles>
</uap:DefaultTile>
```

Adding these attributes may show the tittle.

**Comment by rhaist** [comment]
I applied the fix and uploaded 1.1.4 to the windows store.

REF: https://salsa.debian.org/rhaist-guest/WSL/commit/e579f983143829a324b2bfaa7cd39a3e2b6af044
